From 9e45c8809fc4e0aa1dd2ec2ac3e57352049f1b92 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex@linutronix.de>
Date: Tue, 20 Aug 2024 19:01:59 +0200
Subject: [PATCH] tests/cksuite-all-netns.c: disable route_1

This is a new test added in https://github.com/thom311/libnl/commit/49f7822961f5bc6b18cd2a2d3f3b8d2ab0896d3f
and while it passes on host Debian, it doesn't inside poky.

Investigation into why is proceeding slowly, and I'm simply
not skilled enough in AF_NETLINK, so let's disable until the
core reason for the fail is arrived at.

Upstream-Status: Inappropriate [fail reported and investigated at https://github.com/thom311/libnl/issues/399]
Signed-off-by: Alexander Kanavin <alex@linutronix.de>
---
 tests/cksuite-all-netns.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/cksuite-all-netns.c b/tests/cksuite-all-netns.c
index 5b9d3a5..b2fc46a 100644
--- a/tests/cksuite-all-netns.c
+++ b/tests/cksuite-all-netns.c
@@ -341,7 +341,7 @@ Suite *make_nl_netns_suite(void)
 				  nltst_netns_fixture_teardown);
 	tcase_add_test(tc, cache_and_clone);
 	tcase_add_loop_test(tc, test_create_iface, 0, 17);
-	tcase_add_test(tc, route_1);
+	//tcase_add_test(tc, route_1);
 	suite_add_tcase(suite, tc);
 
 	return suite;
